src/mainTemplate.json

```diff
@@ -82,6 +82,13 @@
         "description": "Install the Plugins - Marvel, Shield, Watcher, Graph* (Elasticsearch 2.3.0+)"
       }
     },
+    "esAdditionalPlugins": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Install additional Plugins. Each plugin separated by a semicolon. e.g. delete-by-query;lmenezes/elasticsearch-kopf/2.0"
```

Comment:
_minor nitpick_ - could we use some other plugins as examples? `delete-by-query` is deprecated in 2.x and replace by delete-by-query API, and Kopf is a third party plugin. For example,

- [ICU Analysis plugin](https://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-icu.html) 
- [Mapper Attachments plugin](https://www.elastic.co/guide/en/elasticsearch/plugins/current/mapper-attachments.html)